flow/designs/gf12/ca53/config.mk

```diff
@@ -8,27 +8,27 @@ export CACHED_NETLIST = $(PLATFORM_DIR)/$(DESIGN_NAME)/rtl/ca53_cpu.v
 export SDC_FILE      = $(PLATFORM_DIR)/$(DESIGN_NAME)/sdc/ca53_cpu.sdc
 
 
-export ADDITIONAL_LEFS += $(PLATFORM_DIR)/lef/sc9mcpp84_12lp_base_lvt_c14.lef
-export ADDITIONAL_LIBS += $(PLATFORM_DIR)/lib/sc9mcpp84_12lp_base_lvt_c14_tt_nominal_max_0p80v_25c.lib
 export ADDITIONAL_GDS  += $(PLATFORM_DIR)/gds/sc9mcpp84_12lp_base_lvt_c14.gds2
```

Comment:
Why are the lvt cells removed (but not their gds)?  This seems unrelated to wrapping.

Reply:
They are not removed, they are just added later in the config file.

Reply:
In that case why not move the gds too?

Reply:
sure